include/astra_camera/astra_driver.h

```diff
@@ -177,6 +180,12 @@ class AstraDriver
   bool use_device_time_;
 
   Config old_config_;
+
+  // Service to enable/disable streaming
+  ros::ServiceServer enable_streaming_srv_;
+  std::string enable_streaming_srv_name_;
+  bool enable_streaming_;
```

Comment:
It might be best to use different `enable_streaming_` for the RGB, IR and Depth streams.